### PR TITLE
Use icons at Header/ Footer, this fixes #10

### LINKED ISSUE #10: Use icons
https://trello.com/c/tJV4zea6/18-icons

### PR DESCRIPTION
**Pull request by AsukaCHikaru**
Using https://github.com/mui-org/material-ui/tree/master/packages/material-ui-icons
This fixes #10 
Preview: 
issue#10

### CODE REVIEW

File: src/style/CreateButton.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    height: 100%;
+    width: 100%;
+    box-shadow: 1px 3px 3px 1px #8888;
+
+    &:hover{
+      background-color: #888;
+    }
+  }
+  
+}
```
Comment (dwi2) [inline src/style/CreateButton.css]: nits: new line here

File: src/style/Footer.css [hunk 2]
```diff
@@ -7,19 +7,25 @@ footer{
...
         background-color: #aaa;
       }
+      & svg{
+        display: block;
+        margin: 0 auto;
+        margin-top: 5px;
+      }
     }
   }
 }
```
Comment (dwi2) [inline src/style/Footer.css]: ditto

File: src/style/Header.css [hunk 3]
```diff
@@ -21,8 +21,12 @@ header{
...
     & button{
+      color: #000;
       width: 100%;
       height: 100%;
+      &:active{
+        color: #aaa;
+      }
     }
   }
 }
```
Comment (dwi2) [inline src/style/Header.css]: ditto

Comment (AsukaCHikaru) [reply src/style/Header.css]: Not quite understand here tho

Comment (dwi2) [reply src/style/Header.css]: Use_icons_at_Header__Footer__this_fixes__10_by_AsukaCHikaru_·_Pull_Request__13_·_project-yori_yori-view
This needs an extra line feed

**Comment by AsukaCHikaru** [comment]
Updated

**Pull request merged**